### ISSUE TITLE
"go get" doesn't install import of httpmock?

### ISSUE THREAD

**Issue by sbwong** [jarcoal/httpmock#36]
When the Go code contains an import statement, e.g.

import {
   ...other packages....
   "gopkg.in/jarcoal/httpmock.v1"
}

and then "go get" is used to install the imports from the whole codebase, the httpmock package is not installed and no error message is produced.  All other imported packages are installed properly however.

"go get gopkg.in/jarcoal/httpmock.v1" works fine.

The same problem occurs for the older github/jarcoal/httpmock version.  We have seen this problem on multiple machines.

On my particular machine, GOROOT, GOPATH (single directory) and GOBIN (= GOPATH + "\bin") environment variables are all set.

Are there any other variables or other specific Go setup that needs to be done in order for "go get" to properly install the httpmock package or is there a bug in the httpmock repository system somewhere?

Thank you!

**Comment by dlebech** [comment]
@sbwong That's where strange :thinking: The library has remained unchanged for many months and has no external dependencies, so it's weird that this would happen now. Just for good measure, I added Golang 1.9 and 1.10 to the travis configuration in #37 , but all the existing tests run just fine without modification. I've used httpmock recently [over here](https://github.com/getconversio/go-shopify) as well without the problem you are describing.

Sorry for not having a specific answer to this. I hope someone else might have an idea.
